### PR TITLE
Added Error handling for improper use of commands. Added fun easter e…

### PR DESCRIPTION
**Pull request by MSaimBeg**
Added Error handling for improper use of commands. Added fun easter egg cog and commands to retain user attention. (Lead: msaimbeg)

### CODE REVIEW

File: cogs/cog_easteregg.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    async def guessing_game(self, ctx):
+        emojis = ["1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"]
+        used = []
+        end_game_flag = True  # manages game loop
+        final_message_flag = True  # makes sure eng game message is
+        attempt_limit = 5
+        attempts = attempt_limit
+        choice = random.choice(emojis)
+        title = "Guess the number!"
+        description = f"Wait for all emojis to be added before reacting/selecting. {choice}"
```
Comment (nattapi) [inline cogs/cog_easteregg.py:156]: Displaying answer to user, should remove choice.

File: cogs/cog_easteregg.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+    @commands.command(name="guessing-game", aliases=["guess", "gg"])  # command to create a note
+    async def guessing_game(self, ctx):
+        emojis = ["1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"]
+        used = []
+        end_game_flag = True  # manages game loop
+        final_message_flag = True  # makes sure eng game message is
+        attempt_limit = 5
+        attempts = attempt_limit
+        choice = random.choice(emojis)
```
Comment (nattapi) [inline cogs/cog_easteregg.py:154]: Should use a more descriptive name for the correct option.

File: cogs/cog_easteregg.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            embed = discord.Embed(title="Timeout Error!!!",
+                                  description="took more than 60 seconds to respond.",
+                                  color=cfg.colors.TIMEOUT)
+            embed.set_thumbnail(url="https://thumbs.dreamstime.com/b/error-icon-16125237.jpg")
+            await ctx.send(embed=embed)
+
+    @commands.command(name="guessing-game", aliases=["guess", "gg"])  # command to create a note
+    async def guessing_game(self, ctx):
+        emojis = ["1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"]
+        used = []
```
Comment (nattapi) [inline cogs/cog_easteregg.py:149]: What is the purpose of used? Should add a more descriptive name.

File: cogs/cog_easteregg.py [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+                color = cfg.colors.SUCCESS
+                title = "Yay! you won."
+
+            # output embed
+            embed = discord.Embed(title=title,
+                                  description=f"Bot: {emoji}\nYou: {reaction.emoji}",
+                                  color=color)
+            embed.set_author(name=ctx.author.display_name, icon_url=ctx.author.avatar_url)
+            embed.set_thumbnail(
+                url=img)
```
Comment (nattapi) [inline cogs/cog_easteregg.py:136]: Is the variable title in line 128 the same variable being assigned in line 131 to title? Having a hard time differentiating them.

File: cogs/cog_easteregg.py [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+            # output embed
+            embed = discord.Embed(title="Magic 8-Ball:", color=colour)
+            embed.set_author(name=ctx.author.display_name, icon_url=ctx.author.avatar_url)
+            embed.set_thumbnail(
+                url="https://external-content.duckduckgo.com/iu/" +
+                    "?u=https%3A%2F%2Fwww.bigw.com.au%2Fmedias%2Fsys" +
+                    "_master%2Fimages%2Fimages%2Fhd5%2Fh14%2F13660536602654.jpg&f=1&nofb=1")
+            embed.add_field(name=response, value="Statement: " + question, inline=True)
+            await ctx.send(embed=embed)
```
Comment (nattapi) [inline cogs/cog_easteregg.py:62]: Does this display the embed to the user? Should add comment.

**Review by nattapi — CHANGES_REQUESTED** [review]
Overall code is written in a very clearly and cleanly way.

File: cogs/cog_easteregg.py (hunk 5 above)
Comment (MSaimBeg) [reply cogs/cog_easteregg.py:62]: I understand, I will clear it up

File: cogs/cog_easteregg.py (hunk 4 above)
Comment (MSaimBeg) [reply cogs/cog_easteregg.py:136]: Yes they are, I will change the assigned title to custom_title

File: cogs/cog_easteregg.py (hunk 3 above)
Comment (MSaimBeg) [reply cogs/cog_easteregg.py:149]: Yeah, I agree. It could be renamed to be more descriptive.

File: cogs/cog_easteregg.py (hunk 2 above)
Comment (MSaimBeg) [reply cogs/cog_easteregg.py:154]: Definetly! choice isn't clear. I'll name is secret_answer_emoji for clarification

File: cogs/cog_easteregg.py (hunk 1 above)
Comment (MSaimBeg) [reply cogs/cog_easteregg.py:156]: Oops! good catch. wouldn't be much of a game then. Will remove!

**Review by MSaimBeg — COMMENTED** [review]
I checked over the change requests, I'll fix them soon.

**Pull request merged**